### PR TITLE
[IMPAC-855] - Include Forecast Payroll img for chart legend

### PR DESCRIPTION
**Pull request by agranado2k**
It depend on [IMPAC-815]

EDIT: IMPAC-815 merged with #505

### CODE REVIEW

File: src/components/widgets-settings/forecast-payroll/forecast-payroll.directive.coffee [hunk 1]
```diff
@@ -0,0 +1,50 @@
+module = angular.module('impac.components.widgets-settings.forecast-payroll',[])
+module.controller('SettingForecastPayrollCtrl', ($scope, $log, ImpacDashboardsSvc, ImpacMainSvc, ImpacWidgetsSvc) ->
```
Comment (xaun) [inline src/components/widgets-settings/forecast-payroll/forecast-payroll.directive.coffee]: D.I `ImpacMainSvc`, `ImpacWidgetsSvc`, and `$log` not used

File: src/components/widgets-settings/forecast-payroll/forecast-payroll.directive.coffee [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  setting.key = "forecast_payroll"
+  setting.isInitialized = false
+
+  # initialization of selected organizations
+  setting.initialize = ->
+    ImpacDashboardsSvc.load().then(
+      (config) ->
+        currentDashboard = config.currentDashboard
+        return unless w.metadata? && w.metadata.forecast_payroll?
+        w.selectedForecastPayroll = w.metadata.forecast_payroll
```
Comment (xaun) [inline src/components/widgets-settings/forecast-payroll/forecast-payroll.directive.coffee]: Is there a reason you're attaching `selectedForecastPayroll` onto the widget object? I know impac-angular does this across the framework, but that is old architecture and it's not really a good practice. Based on what I can see here, I would just use `$scope.selectedForecastPayroll`.

File: src/components/widgets-settings/forecast-payroll/forecast-payroll.tmpl.html [hunk 3]
```diff
@@ -3,7 +3,7 @@ <h5 translate>impac.widget.settings.forecast_payroll.title</h5>
 
   <div class="widget-lines-container">
     <div class="widget-line">
-      Aggregate
+      {{'impac.widget.account_cash_projection.forecast_payroll.aggregate' | translate}}
```
Comment (xaun) [inline src/components/widgets-settings/forecast-payroll/forecast-payroll.tmpl.html]: @agranado2k when not interpolating a translate alongside other text, prefer this syntax
```html
<div class="widget-line" translate>
  impac.widget.account_cash_projection.forecast_payroll.aggregate
</div>
```

**Comment by cesar-tonnoir** [comment]
Merging
@xaun can't change the syntax as it makes the toggle button disappear...

**Pull request merged**